src/librbd/image/RefreshParentRequest.h

```diff
@@ -4,7 +4,6 @@
 #ifndef CEPH_LIBRBD_IMAGE_REFRESH_PARENT_REQUEST_H
 #define CEPH_LIBRBD_IMAGE_REFRESH_PARENT_REQUEST_H
 
-#include "include/int_types.h"
```

Comment:
@cxwshawn I personally don't like that logic since you are introducing new inter-dependencies instead of just cleaning out unnecessary includes.